### DISCUSSION TITLE
Is it possible to run a single dagit UI instance and have multiple docker containers connect to it?

### DISCUSSION THREAD

**Discussion by dagsir** [dagster-io/dagster#17537] [Q&A]
Is it possible to run a single dagit UI instance and have multiple docker containers connect to it?

---

*The question was originally asked in [Dagster Slack](https://dagster.slack.com/archives/C01U954MEER/p1698774558816029?thread_ts=1698774558.816029&cid=C01U954MEER).*

**Comment by shalabhc** [comment]
If you mean multiple independent instances of dagster connect to the same UI - that is not supported.

**Reply by benwenger-xplor** [reply]
We want multiple containers each with their own repo that point to a singular container running the UI. is that still not supported?
